packages/next/pages/_document.tsx

```diff
@@ -518,6 +519,11 @@ export class Head extends Component<
           </>
         )}
         {React.createElement(React.Fragment, {}, ...(headTags || []))}
+        <script
```

Comment:
This should be the very first `<script>` after `<link rel=preload>` but before the first `<script defer/async>`.

h/t @developit

Reply:
Created a helper render method to make sure it's before the only async app script (in AMP mode) 👍